pkg/kritis/signer/signer.go

```diff
@@ -75,13 +75,19 @@ func (s Signer) ValidateAndSign(imageVulnz ImageVulnerabilities, vps v1beta1.Vul
 	}
 
 	glog.Infof("Image %q passes VulnzSigningPolicy %s.", imageVulnz.ImageRef, vps.Name)
-	existed, _ := s.isAttestationAlreadyExist(imageVulnz.ImageRef)
+	return s.SignImage(imageVulnz.ImageRef)
+}
+
+// ValidateAndSign signs an image without doing any policy check.
+// Returns an error if image does not pass or creating an attestation fails.
+func (s Signer) SignImage(image string) error {
+	existed, _ := s.isAttestationAlreadyExist(image)
```

Comment:
Is it safe to ignore this error?  Shouldn't block this PR, but let's think about it.